packages/table/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blueprintjs/table",
-  "version": "3.3.0",
+  "version": "3.3.0-graphext1",
```

Comment:
Is this the final version ?

I've seen a `3.3.0-graphext2` in https://github.com/graphext/graphext/pull/261/files

Reply:
Yep.. it's the `-graphext2`, and after the latest changes if will be `-graphext3`

Reply:
cool